src/coreclr/tools/aot/crossgen2/crossgen2.props

```diff
@@ -15,9 +15,11 @@
     <EnableDefaultEmbeddedResourceItems>false</EnableDefaultEmbeddedResourceItems>
     <Configurations>Debug;Release;Checked</Configurations>
     <RunAnalyzers>false</RunAnalyzers>
+    <UseLocalTargetingRuntimePack Condition="'$(StageTwoBuild)' == 'true'">true</UseLocalTargetingRuntimePack>
```

Comment:
Can this be in `toolAot.targets` so that it applies to all tools published with Aot automatically?

Reply:
We can try :)

```suggestion
```

Reply:
Unfortunately, it is giving errors like `/__w/1/s/src/libraries/Directory.Build.targets(120,3): error MSB4011: "/__w/1/s/eng/targetingpacks.targets" cannot be imported again. It was already imported at "/__w/1/s/eng/toolAot.targets (7,3)". This is most likely a build authoring error. This subsequent import will be ignored. [/__w/1/s/src/libraries/System.Runtime.InteropServices/gen/Microsoft.Interop.SourceGeneration/Microsoft.Interop.SourceGeneration.csproj] [/__w/1/s/Build.proj]`

We will need to add tracking. targetingpacks.targets is used in 29 places. I'm leaving it as is for now.